x-pack/plugins/maps/common/i18n_getters.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export function getAppTitle() {
+  return i18n.translate('xpack.maps.appTitle', {
+    defaultMessage: 'Maps'
+  });
+}
+
+
+export function getDataSourceLabel() {
+  return i18n.translate('xpack.maps.source.common.dataSourceLabel', {
```

Comment:
Not sure you need to add `.common` to the id. `xpack.maps.source.dataSourceLabel` makes more sense and can be easily found with a search